sdk/react/panels/EndScreen.js

```diff
@@ -69,10 +69,11 @@ var EndScreen = React.createClass({
   },
 
   _renderDefaultScreen: function(progressBar, controlBar) {
-    var fullscreenPromoImage = (this.props.config.endScreen.mode == 'default');
+    var endScreenConfig = this.props.config.endScreen || {};
```

Comment:
it is always an object, and thx for catching the error `endScreenConfig === 'default'` :)